Inspicio/wwwroot/js/quickCreate.js

```diff
@@ -91,10 +91,13 @@
             __RequestVerificationToken: $('[name= "__RequestVerificationToken"]').val(),
             CreatePageModel: CreatePageModel
         }
+        var controllers = ["Images", "Account", "Images", "Manage"];
+        var path = location.pathname.split("/");
+        alert("http://" + location.host + path[1] + "/Images/Create");
```

Comment:
This looks like it was commited by mistake, I'm assuming it's supposed to do something with the controllers array, and that the alert is only a debug thing?